apps/package.json

```diff
@@ -61,7 +61,7 @@
     "lodash-cli": "~2.4.2",
     "messageformat": "^0.2.1",
     "mkdirp": "^0.5.0",
-    "mochify": "^2.8.1",
+    "mochify": ">=2.8.1 <2.10",
```

Comment:
Just documenting: Elijah discovered that certain tests (in particular ones that load and render ejs templates) were breaking with mochify >= 2.10.